sample/App.js

```diff
@@ -151,36 +152,46 @@ const App: () => React$Node = () => {
             <View style={styles.sectionContainer}>
               <Text
                 style={styles.sectionTitle}
+                sentryID="captureMessage"
```

Comment:
I went through existing ids with @HazAT, there were `nativeID` and `testID` but they wouldn't exactly match the purpose we need, along with potentially having side effects: 
<img width="794" alt="Screen Shot 2020-06-18 at 12 20 17 AM" src="https://user-images.githubusercontent.com/30991498/84929046-7edd5080-b0f9-11ea-8229-317982cdf4b7.png">

Other than those two, there is `accessibilityLabel` which we would track if present, but I chose to have a unique id because `accessibilityLabel` is used to label the uses of that element for screen readers is not an id: https://reactnative.dev/docs/accessibility#accessibilitylabel-android-ios

Reply:
Oh and almost forgot to note that as `e._targetInst` is not documented, I had to find this method of finding the React that received the touch by attaching a debugger and looking through the event object manually. I could not find any way to get the name of the component/element, even though we can get the id of the element, they removed the method to get the component from its id two years ago: `ReactNativeComponentTree`. The id in this case is an arbitrary number that would have no value to the user on its own.

Reply:
Would be nice to have a test that makes sure `e._targetInst` is there so we have a breaking CI if we try to bump to a breaking version of RN. That way we have a choice of fixing things, staying at an older version or else. As opposed to accidentally releasing a broken integration.

Reply:
Yes I agree, hmm it can be tested as a part of the e2e tests.